config/locales/de.application_letters.yml

```diff
@@ -76,9 +76,12 @@ de:
     pending_after_deadline: "In Bearbeitung"
     alternative: "Nachrücker"
 
-
   controllers:
     application_letters:
       confirmation_mail:
         subject: "Bewerbung erhalten"
-        content: "Hallo, \n\rDu hast dich erfolgreich für  {{semimar_name}} beworben. Deine Bewerbung wird jetzt bearbeitet. Für weitere Information besuche das Workshop Portal. Viele Grüße, das HPI Workshop Team"
+        content: >
+          Liebe(r) {{first_name}} {{last_name}},\n\r
```

Comment:
@julisa99 just requested different greetings according to gender ("Lieber" for male, "Liebe" for female, "Liebe(r)" for other). Could you do that?